tasklist/client/e2e/tests/task-details.spec.ts

```diff
@@ -177,8 +178,17 @@ test.describe('task details page', () => {
     await expect(tasksPage.completeTaskButton).not.toBeVisible();
 
     await tasksPage.openTask('JobWorker_user_task');
-    await expect(page.getByText('jobWorkerVar')).toBeVisible();
-    await expect(page.getByText('zeebeVar')).toBeVisible();
+
+    // this is necessary because sometimes the importer takes some time to receive the variables
```

Comment:
In Optimize, we have a wait step for the import to complete.
We used to have an endpoint for that but for C8, i currently have curl script that checks elasticsearch directly if all the test data are imported.